.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-    "files.eol": "\n",
-    "typescript.tsdk": "node_modules/typescript/lib"
+	"files.eol": "\n",
```

Comment:
?

Reply:
It's the linter that formats it like that during commit.